### DISCUSSION TITLE
Request for reference result of Project 1 Part 2

### DISCUSSION THREAD

**Discussion by NextBG** [sist-cs120/project-wiki#14] [Project 1 Q&A] [answered]
I wonder whether we can get a reference spectrum of Project1 Part2, to check the correctness of the output?
![image](https://user-images.githubusercontent.com/49566324/136687062-51189665-da04-44fc-9ce2-3b3e3e09960a.png)

**Comment by yhyan96** [comment, marked as answer]
The APPs listed in Tip3 of Part 2 are useful. I use FFT Spectrum Analyzer and the reference result may like that,
![Screenshot_2021-10-10-22-51-13-522_com xyz fft](https://user-images.githubusercontent.com/37173977/136701040-c2e87ed4-9826-4a3f-bbe4-fa88ac39b775.jpg)

If the peak is not such sharp, please consider the update of the phase.
